diff --git a/lib/ber/writer.js b/lib/ber/writer.js
index 3515acf7940ef3ddcfb214fb0e8889e678d19e5d..0e9cae4e13514ed1980422a4370c6e723f812376 100644
--- a/lib/ber/writer.js
+++ b/lib/ber/writer.js
@@ -161,7 +161,7 @@ Writer.prototype.writeBuffer = function (buf, tag) {
 
 
 Writer.prototype.writeStringArray = function (strings) {
-  if ((!strings instanceof Array))
+  if (!(strings instanceof Array))
     throw new TypeError('argument must be an Array[String]');
 
   var self = this;
